<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_product_degree</class>
 <widget class="QDialog" name="new_product_degree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="DegreeTableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>851</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <column>
    <property name="text">
     <string>Grade</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Grupo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Subgrupo</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="degree_lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>59</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Grade</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="group_lineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>59</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Grupo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subgroup_lineEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>59</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Subgrupo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_pushButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>131</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="register_pushButton">
   <property name="geometry">
    <rect>
     <x>707</x>
     <y>320</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar grade</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
